src/snowflake/snowpark/table.py

```diff
@@ -273,9 +273,9 @@ def __init__(
         table_name: str,
         session: Optional["snowflake.snowpark.session.Session"] = None,
         ast_stmt: Optional[proto.Assign] = None,
-        suppress_ast: bool = False,
```

Comment:
nit: same here, should be public. tbh, I am ok to call it always `_emit_ast`. Will make things easier.

Reply:
Changed all files to use `_emit_ast` uniformly.